atd-toolbox/data_model/python/column_metadata.json

```diff
@@ -1359,7 +1600,23 @@
     "record_type": "crashes",
     "source": "cris"
   },
-
+  {
+    "column_name": "geom",
+    "data_type": "GEOMETRY(Point, 4326)",
+    "data_type_constraint": "GENERATED ALWAYS AS (ST_SetSRID(ST_MakePoint(longitude, latitude), 4326)) STORED",
+    "foreign_column_cascade": "",
+    "foreign_column_name": "",
+    "foreign_table_name": "",
+    "foreign_table_schema": "",
+    "is_cris_column": false,
+    "is_edits_column": false,
+    "is_existing_foreign_table": false,
+    "is_primary_key": false,
+    "is_unified_column": true,
+    "note": "",
+    "record_type": "crashes",
+    "source": "cris"
```

Comment:
what exactly does `source` mean here? i would think "db generated field" would be more fitting which i see as the source for some other generated columns

Reply:
thanks for catching this—i will fix this. this property gets translated to the `is_imported_from_cris` column in the `_column_metadata_table`, which in turn is used to generate the upsert mutations in the CRIS import ETL.

so we can set this to any value we want, except `cris`, and it will be properly ignored by the ETL.

it will be nice when we don't to detail with this JSON and the spaghetti migration-gen scripts anymore 😅